### ISSUE TITLE
phpmyadmin container is showing unhealthy status

### ISSUE THREAD

**Issue by rfay** [drud/ddev#448]
## What happened (or feature request):

ddev 0.9.1:
I started several sites; `docker ps` for all of them showed "unhealthy" status for phpmyadmin container. No indication of any problem was given by `ddev list`

`d4cbdb4c87ef        drud/phpmyadmin:v0.2.0             "/run.sh phpmyadmin"     5 minutes ago       Up 5 minutes (unhealthy)       0.0.0.0:32811->80/tcp`

I *was* able to hit phpmyadmin on the URL provided by `ddev describe`, and got the right database.

So we probably have a couple of problems:
* Docker status for the phpmyadmin container isn't reporting correctly
* We're not testing docker status on phpmyadmin for ddev's status checks.

## Related source links or issues:

**Comment by rfay** [comment]
Just checked, this is completely repeatable. I believe it happens for every site everywhere. Start a site. Wait a couple of minutes, do a `docker ps`.

**Comment by rfay** [comment]
* The actual breakage here was https://github.com/drud/docker.phpmyadmin/pull/4 (already resolved, but the ddev image tag for phpmyadmin was never bumped) - Brad and I chased it because this wasn't happening to him. 
* It seems I had built a local image tagged v0.2.0 (just checking out upstream/master when it was at the v0.2.0 tag and `make container` would do this). So I had the container tagged v0.2.0 with the problem listed in the PR.

TODO:
- [ ] ddev needs to check all containers' docker health. Seems not to be checking phpmyadmin. This can be tested by just breaking the healthcheck anyway we want and seeing if ddev notices.
- [ ] We need to do a release on phpmyadmin and bump the ddev version for drud/phpmyadmin.
- [ ] We probably need a test in ddev that makes sure that all container health is checked (not sure)

Thanks for playing on this Brad. Wish we could both remember what happened like last week. Or at least that *you* could.
